js/index.js

```diff
@@ -44,27 +44,98 @@ function renderPepperoni() {
 
 function renderMushrooms() {
   // Iteration 1: set the visibility of `<section class="mushroom">`
+  document.querySelectorAll('.mushroom').forEach(oneMush => {
+    if (state.mushrooms) {
+      oneMush.style.visibility = 'visible';
+    } else {
+      oneMush.style.visibility = 'hidden';
+    }
+  });
 }
 
 function renderGreenPeppers() {
   // Iteration 1: set the visibility of `<section class="green-pepper">`
+  document.querySelectorAll('.green-pepper').forEach(oneGP => {
+    if (state.greenPeppers) {
+      oneGP.style.visibility = 'visible';
+    } else {
+      oneGP.style.visibility = 'hidden';
+    }
+  });
 }
 
+
 function renderWhiteSauce() {
   // Iteration 2: add/remove the class "sauce-white" of `<section class="sauce">`
+  let sauce = document.querySelector('.sauce');
+    if (state.whiteSauce) {
+      sauce.classList.add('sauce-white')
+    } else {
+      sauce.classList.remove('sauce-white")
+    }
 }
+//classlist toggle + classlist remove (w classlist add)
 
 function renderGlutenFreeCrust() {
   // Iteration 2: add/remove the class "crust-gluten-free" of `<section class="crust">`
+  let crust = document.querySelector('.crust');
+    if (state.glutenFreeCrust) {
+      crust.classList.add('crust-gluten-free');
+    } else {
+      crust.classList.remove('crust-gluten-free');
+    }
 }
 
 function renderButtons() {
   // Iteration 3: add/remove the class "active" of each `<button class="btn">`
+let pepBtn = document.querySelector('.btn-pepperoni');
+let mushBtn = document.querySelector('.btn-mushrooms')
+let greenPepBtn = document.querySelector('.btn-green-peppers')
+let gfBtn = document.querySelector('.btn-crust')
+let whiteSauceBtn = document.querySelector('.btn-sauce') 
+
+if (state.pepperoni) {
```

Comment:
ideally you could have done this part with a much more "DRY" approach. but this totally works!